spec/presenters/claim/base_claim_presenter_spec.rb

```diff
@@ -323,20 +328,27 @@
     let(:defendant_added_second) do
       build(:defendant).tap do |defendant|
         travel_to 2.days.ago do
-          defendant.representation_orders = [build(:representation_order, representation_order_date: Date.new(2015, 3, 1), maat_reference: '444444')]
+          defendant.representation_orders = [build(:representation_order,
+                                                   representation_order_date: Date.new(2015, 3, 1),
+                                                   maat_reference: '444444')]
         end
       end
     end
 
     it 'returns a string of all the dates' do
-      expect(presenter.representation_order_details).to eq('01/03/2015 222222<br>13/08/2015 333333<br>01/03/2015 444444')
+      expect(presenter.representation_order_details).to eq(
+        '01/03/2015 222222<br>13/08/2015 333333<br>01/03/2015 444444'
+      )
     end
   end
 
-  it '#case_worker_names' do
-    claim.case_workers << build(:case_worker, user: build(:user, first_name: 'Alexander', last_name: 'Bell'))
-    claim.case_workers << build(:case_worker, user: build(:user, first_name: 'Louis', last_name: 'Pasteur'))
-    expect(subject.case_worker_names).to eq('Alexander Bell, Louis Pasteur')
+  describe '#case_worker_names' do
+    before do
+      claim.case_workers << build(:case_worker, user: build(:user, first_name: 'Alexander', last_name: 'Bell'))
+      claim.case_workers << build(:case_worker, user: build(:user, first_name: 'Louis', last_name: 'Pasteur'))
+    end
+
+    it { expect(presenter.case_worker_names).to eq('Alexander Bell, Louis Pasteur') }
```

Comment:
`claim.case_workers` appears to be empty when `claim` is first created but it is possible that the factory method could change resulting in this test breaking. It might be worth explicitly setting it to ensure that this does not happen.

```suggestion
  describe '#case_worker_names' do
    before do
      claim.case_workers = [
          build(:case_worker, user: build(:user, first_name: 'Alexander', last_name: 'Bell')),
          build(:case_worker, user: build(:user, first_name: 'Louis', last_name: 'Pasteur'))
        ]
    end

    it { expect(presenter.case_worker_names).to eq('Alexander Bell, Louis Pasteur') }
```

There are other examples of this throughout the file.